enhancements/dns/plugins.md

```diff
@@ -267,25 +290,25 @@ metrics are exported. Ensure these metrics are surfaced through the OpenShift mo
 
 #### Gap 4
 
-An increased utilization of compute resources on each node due to the potential of running many `ForwardPlugins`, each
-with up to 15 nameservers that are actively checked for health.
+An increased utilization of compute resources on each node due to the potential of running many `Servers`, each with up
+to 15 upstreams being actively checked for health.
 
 #### Mitigation 4
 
-Test utilization using multiple `ForwardPlugins` with multiple `Nameservers`. Add a warning in the documentation that
-adding a large number of forwarders may incur a performance penalty or hit memory limits.
+Test utilization using multiple `Servers` with multiple `Upstreams`. Add a warning in the documentation that adding a
+large number of forwarders may incur a performance penalty or hit memory limits.
```

Comment:
Also, the operator could increase the memory request based on the number of servers.

Reply:
ack